pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -0,0 +1,66 @@
+package ovirt
```

Comment:
I didn't realize that the install-config yaml for ovirt has the credentials inlined.. 
This is not safe :( 
we should be using the env based sources for fetching the creds :(

Reply:
fwiw we discussed this before to some extent: https://github.com/openshift/installer/pull/1948#discussion_r339841555

Reply:
Yeah, we should have implemented your proposal before merging, but i wouldn't block this PR on this change now :(